data/controller_archive.yaml

```diff
@@ -1,4 +1,7 @@
 # Archive pages for Controller
+v_4_3:
```

Comment:
```suggestion
v_4_3_0:
```

Reply:
Waiting for @tvo318 to confirm. When I follow the links, it isn't 4.3.0, it's 4.3 - https://docs.ansible.com/automation-controller/4.3/html/userguide/

Reply:
@oraNod right, I haven't published them yet.

Reply:
@oraNod is it ok to drop the .0 designation forever?